libs/core/src/lib/list/list.component.ts

```diff
@@ -143,7 +147,9 @@ export class ListComponent implements OnInit, AfterContentInit, OnDestroy {
     ngAfterContentInit(): void {
```

Comment:
setting focus on first list item in `ngAfterContentInit` hook is not correct.
if you go to https://sap.github.io/fundamental-ngx/#/core/list. here automatically `first list item` of [last list](https://sap.github.io/fundamental-ngx/#/core/list#infinite-scroll) is getting focused; without trying to focus on this list. which is wrong.
This should be fixed in List, then automatically combo-box will have correct behaviour.